.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
-  - "10"
+  - '10'
 
-script: npm run verify
+
+  script:
```

Comment:
Your yaml is unparsable because script isn't at the beginning of the line here. That's why Travis isn't running.

Reply:
Hmm, this was valid when I looked at their docs, wonder why it's not working here
:(

Reply:
Oh I see you want only one CI script to run

Reply:
Yes. It's not a travis problem, your yaml syntax is just unparseable. YAML is indentation sensitive. Try pasting what you've written into, say, https://yaml-online-parser.appspot.com/ and you'll see what I mean

Reply:
I think my change fixes this. Thank you :)

Reply:
Oops lol I used Groovy with Jenkins so this is a bit different than what I am use to lol